pkg/rancher-desktop/pages/containers/ContainerInfo.vue

```diff
@@ -0,0 +1,385 @@
+<template>
+  <div
+    class="container-info-page"
+    data-testid="container-info"
+  >
+    <div class="tab-header-row">
+      <ul class="tabs">
+        <li
+          class="tab active"
+          data-testid="tab-logs"
+        >
+          <a>
+            <span>Logs</span>
+          </a>
+        </li>
+      </ul>
+      <div class="search-widget">
+        <input
+          ref="searchInput"
+          v-model="searchTerm"
+          aria-label="Search in logs"
+          class="search-input"
+          data-testid="search-input"
+          placeholder="Search logs..."
+          type="search"
+          @input="onSearchInput"
+          @keydown="handleSearchKeydown"
+        >
+        <button
+          :disabled="!searchTerm"
+          aria-label="Previous match"
+          class="search-btn btn role-tertiary"
+          data-testid="search-prev-btn"
+          title="Previous match"
+          @click="searchPrevious"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-chevron-up"
+          />
+        </button>
+        <button
+          :disabled="!searchTerm"
+          aria-label="Next match"
+          class="search-btn btn role-tertiary"
+          data-testid="search-next-btn"
+          title="Next match"
+          @click="searchNext"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-chevron-down"
+          />
+        </button>
+        <button
+          :disabled="!searchTerm"
+          aria-label="Clear search"
+          class="search-btn btn role-tertiary"
+          data-testid="search-clear-btn"
+          title="Clear search"
+          @click="clearSearch"
+        >
+          <i
+            aria-hidden="true"
+            class="icon icon-x"
+          />
+        </button>
+      </div>
+    </div>
+    <div class="tab-content">
+      <container-logs
+        v-if="containerId"
+        ref="containerLogs"
+        :container-id="containerId"
+        :is-container-running="isRunning"
+        :namespace="settings?.containers?.namespace"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
```

Comment:
Oh, nice, you converted to composition API too. (The rest of the code base is still in options API.)